cnxarchive/tests/test_views.py

```diff
@@ -3051,3 +3051,18 @@ def patched_now_side_effect(timezone):
         expected_file = os.path.join(testing.DATA_DIRECTORY, 'robots.txt')
         with open(expected_file, 'r') as f:
             self.assertMultiLineEqual(robots, f.read())
+
+    def test_recent_rss(self):
+        self.request.matched_route = mock.Mock()
+        self.request.matched_route.name = 'recent'
+
+        from ..views import recent
+        recent = recent(self.request)
+        # # note: recent is the dict that is returned by recent function
+
+        self.assertEqual(len(recent['latest_modules']), 10)
+        for module in recent['latest_modules']:
+            keys = module.keys()
+            keys.sort()
+            self.assertEqual(keys,
+                             ["abstract", "authors", "link", "name", "revised"])
```

Comment:
This is a good start. 👍  You may also want to test that you are actually getting the correctly ordered results.